### PR TITLE
[MNOE-501] minor-UI-change-on-boarding

### PR DESCRIPTION
**Pull request by rheasunshine**
@ouranos @alexnoox 
Screenshots in JIRA.. 
Pass or Fail?

### CODE REVIEW

File: src/app/components/mno-app-onboarding-btn/mno-app-onboarding-btn.html [hunk 1]
```diff
@@ -9,8 +9,8 @@
     <span>{{$ctrl.conflictingApp.name}}</span>
   </div>
   <div ng-switch-when="INSTALLED_LAUNCH">
-    <i class="fa fa-check fa-2x button-check"></i><br>
-    <span class="text-muted">(connected)</span>
+    <span>No set-up required</span><br>
+    <span class="text-muted">This app connects when launched.</span>
```
Comment (alexnoox) [inline src/app/components/mno-app-onboarding-btn/mno-app-onboarding-btn.html]: Please add those text string into locales keys.
(I forgot to i18n the `(connected)`)

**Comment by alexnoox** [comment]
Please remove the impac locales files.

**Comment by rheasunshine** [comment]
@alexnoox made these changes. Let me know if there's anything else. 8-)

**Comment by alexnoox** [comment]
@rheasunshine Thanks, all good now.
Please new update the locales in https://github.com/maestrano/mno-enterprise/tree/3.3/core/config/locales

**Comment by rheasunshine** [comment]
Hey @alexnoox , I just changed a few of the locales' names to match mno-enterprise

**Comment by rheasunshine** [comment]
Pull request made for locales in mno-enterprise

**Comment by ouranos** [comment]
Locales at maestrano/mno-enterprise#333

**Comment by alexnoox** [comment]
LGTM

**Pull request merged**